jobs/future-effective-filings/knative/dc.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: $NAME
+  generation: $GENERATION
+  labels:
+    cloud.googleapis.com/location: northamerica-northeast1
+spec:
+  template:
+    metadata:
+      labels:
+        run.googleapis.com/startupProbeType: Default
+      annotations:
+        run.googleapis.com/client-name: cloud-console
+        run.googleapis.com/cloudsql-instances: $CLOUDSQL_INSTANCES
+        autoscaling.knative.dev/maxScale: $MAXSCALE
+    spec:
+      containerConcurrency: $CONTAINER_CONCURRENCY
+      timeoutSeconds: $TIMEOUT_SECONDS
+      serviceAccountName: $SERVICE_ACCOUNTNAME
+      containers:
+      - image: >-
+          $CONTAINER_IMAGE
+        ports:
+        - name: http1
+          containerPort: $CONTAINER_PORT
+        env:
+        - name: DATABASE_HOST
+          value: $DATABASE_HOST
+        - name: DATABASE_NAME
+          value: $DATABASE_NAME
+        - name: DATABASE_PASSWORD
+          value: $DATABASE_PASSWORD
+        - name: DATABASE_PORT
+          value: $DATABASE_PORT
+        - name: GUNICORN_PROCESSES
+          value: $GUNICORN_PROCESSES
+        - name: GUNICORN_THREADS
+          value: GUNICORN_THREADS
+        - name: DATABASE_UNIX_SOCKET
+          value: $DATABASE_UNIX_SOCKET
+        - name: DATABASE_USERNAME
+          value: DATABASE_USERNAME
+        - name: AUDIENCE
+          value: $AUDIENCE
+        - name: PUBLISHER_AUDIENCE
+          value: $PUBLISHER_AUDIENCE
+        - name: ENTITY_MAILER_TOPIC
+          value: $ENTITY_MAILER_TOPIC
```

Comment:
may not need as per comment above.

Reply:
def'n needed

Reply:
Thor - i don't see emailer topic being used anywhere in this job. 

Just for my understanding, are we leaving the emailer topic as we may need to use this in the future?  Or is this is dc.yaml specific thing only?

Reply:
I was meaning more of the ENV and YAML files still being needed, until Patrick says we don't

Reply:
ok, gotcha.  @JazzarKarim I think we can remove emailer topic references then.

Reply:
Alright, sounds good. I'm on it.